client/coral-framework/components/Slot.js

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 import cn from 'classnames';
 import styles from './Slot.css';
+import {connect} from 'react-redux';
 import {getSlotElements} from 'coral-framework/helpers/plugins';
 
-export default function Slot ({fill, inline = false, ...rest}) {
+function Slot ({fill, inline = false, plugin_config: {debug, config}, ...rest}) {
```

Comment:
Just saw it in the description. Seems like there have been some thought put into this. Would love to hear about it :-)